packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -367,9 +376,18 @@ const presetPrompt = () => {
     const questions = [
         {
             name: 'preset',
-            message: 'Choose a project preset to get started:',
+            message: 'Choose a project to get started:',
             type: 'list',
-            choices: PUBLIC_PRESETS
+            choices: [
+                {
+                    name: 'The Retail app with demo Commerce Cloud sandbox',
+                    value: RETAIL_REACT_APP_DEMO
+                },
+                {
+                    name: 'The Retail app using your own Commerce Cloud sandbox',
```

Comment:
"Sandbox" doesn't feel like the right word to use here.

Though you can use a sandbox, you might not. You can connect to a production, staging or development instance.

I'd suggest:

 ```suggestion
                    name: 'The Retail app using your own Commerce Cloud instance',
```

Reply:
Sounds good, I committed the suggestion and I'll confirm with @jennmills.